tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -422,20 +406,13 @@ def test_it_sets_the_right_javascript_config_settings(
             }
         )
 
-        self._assert_authorization_valid_jwt(
-            form_fields["authorization"],
-            {
-                "oauth_consumer_key": "TEST_OAUTH_CONSUMER_KEY",
-                "roles": "TEST_ROLES",
-                "user_id": "TEST_USER_ID",
-            },
+        BearerTokenSchema.assert_called_once_with(pyramid_request)
+        bearer_token_schema.authorization_param.assert_called_once_with(
+            pyramid_request.lti_user
         )
-
-    def _assert_authorization_valid_jwt(self, authorization, expected_values):
-        assert authorization.startswith("Bearer ")
         assert (
-            decode_jwt(authorization[len("Bearer ") :], "test_secret")
-            == expected_values
+            form_fields["authorization"]
+            == bearer_token_schema.authorization_param.return_value
```

Comment:
We don't care _what_ the authorization string is (that's `BearerTokenSchema`'s job, and it has its own unit tests) we only care that it came from `BearerTokenSchema`